sorting/quick_sort.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ *      copyright The Algorithms
+ *      Author - 
+ *      Correction - ayaankhan98
+ *
+ */
+
+#include <cstdlib>
+#include <iostream>
+
+int partition(int arr[], int low, int high) {
```

Comment:
please add documentation to the code.

Reply:
okay i will fix that in another PR

Reply:
@kvedala new PR #801

Reply:
Thank you. will follow that now 👍

Reply:
You can mark #801 as draft pull-request and mark it "ready for review" when it is. that way, I will be better able to review it.

Reply:
okay sure